src/graphql/schema.gql

```diff
@@ -210,6 +210,9 @@ input VoteWhere {
   vp_lte: Float
   vp_state: String
   vp_state_in: [String]
+  choice: String
+  choice_in: [String]
+  choice_in_json_keys: [String]
```

Comment:
Instead of a separate variable for objects and seperate variable for strings, can we just check both with one variable?

Reply:
I mean maybe we can handle it outside buildWhereQuery, once we pass a string, it should check both as string and JSON